internalWebhook.go

```diff
@@ -22,8 +22,8 @@ type Register interface {
 }
 
 type RegistryV1 struct {
-	PartnerIDs []string
-	Registration    webhook.RegistrationV1
+	PartnerIDs   []string
+	Registration webhook.RegistrationV1 `json:"Webhook"`
```

Comment:
webhook json tag is needed so the data can be properly unmarshaled into the RegistryV1 struct